recipe/meta.yaml

```diff
@@ -14,14 +14,14 @@ source:
     - cap_mpjpeg_decoder.patch  # [win]
 
 build:
-    number: 0
+    number: 1
 
 requirements:
   build:
     - python
     - toolchain
     # For sha256 comparisons of opencv_contrib
-    - openssl               # [unix]
+    - openssl 1.0.*         # [unix]
```

Comment:
It's what the pinning script would do anyways. No point in causing it to regenerate this needlessly and thus gobble up CI resources.